packages/editor/src/components/widgets/collapsible/Collapsible.tsx

```diff
@@ -22,14 +30,15 @@ const Collapsible = ({ label, defaultOpen, message, children, autoClosable }: Co
   }, [autoClosable, defaultOpen]);
   return (
     <CollapsibleRoot className='collapsible-root' open={open} onOpenChange={setOpen}>
-      <div className='collapsible-header'>
+      <div className={`collapsible-header${!open ? '-closed' : ''}`}>
```

Comment:
I wouldn't change the whole class. Makes it only much harder to test IMO.
Better don't change the default component class, and handle such state stuff in a second class or in a data attribute e.g. `data-state={open ? 'open' : 'closed'}`

And if you do such things, I can only speak for myself, but IMO negative stuff is much harder to read. So even if you don't change this to the attribute style I would write it like `open ? '' : '-closed'`